metadata-db-app/src/cmr/metadata_db/data/oracle/concepts.clj

```diff
@@ -364,10 +365,10 @@
 
 (defn get-concept
```

Comment:
This function is getting a bit long, and is a candidate for being converted to a multi-method based on the state of the dynamo-config. To be functional we should properly pass in the config, as opposed to having a global state object change the behavior of the function.

Reply:
instead of a multi-method could we use cond or condp or case to call other functions?  Or are you being more generic?  I keep reading that multi-methods are much slower.

Reply:
During testing, all parts of this function are actually going to be called at the same time. I'm not aware if multi-methods can make that work. Also, I would like to clarify that the current structure of these functions is for prototype testing purposes, this is not the state it will be in when merged into master